golden_response/staging/get_stat_all/result.json

```diff
@@ -1,715 +1,765 @@
+        "1956":  168903031,
+        "1957":  171984130,
+        "1958":  174881904,
+        "1959":  177829628,
+        "1960":  180671158,
+        "1961":  183691481,
+        "1962":  186537737,
+        "1963":  189241798,
+        "1964":  191888791,
+        "1965":  194302963,
+        "1966":  196560338,
+        "1967":  198712056,
+        "1968":  200706052,
+        "1969":  202676946,
+        "1970":  205052174,
+        "1971":  207660677,
+        "1972":  209896021,
+        "1973":  211908788,
+        "1974":  213853928,
+        "1975":  215973199,
+        "1976":  218035164,
+        "1977":  220239425,
+        "1978":  222584545,
+        "1979":  225055487,
+        "1980":  226542250,
+        "1981":  229465744,
+        "1982":  231664432,
+        "1983":  233792014,
+        "1984":  235824908,
+        "1985":  237923734,
+        "1986":  240132831,
+        "1987":  242288936,
+        "1988":  244499004,
+        "1989":  246819222,
+        "1990":  250131894,
+        "1991":  253492503,
+        "1992":  256894189,
+        "1993":  260255352,
+        "1994":  263435673,
+        "1995":  266557091,
+        "1996":  269667391,
+        "1997":  272911760,
+        "1998":  276115288,
+        "1999":  279294713,
+        "2000":  282162411,
+        "2001":  284968955,
+        "2002":  287625193,
+        "2003":  290107933,
+        "2004":  292805298,
+        "2005":  295516599,
+        "2006":  298379912,
+        "2007":  301231207,
+        "2008":  304093966,
+        "2009":  306771529,
+        "2010":  309326085,
+        "2011":  311580009,
+        "2012":  313874218,
+        "2013":  316057727,
+        "2014":  318386421,
+        "2015":  320742673,
+        "2016":  323071342,
+        "2017":  325147121,
+        "2018":  327167434,
+        "2019":  328239523
        },
-       "measurementMethod": "CensusPEPSurvey",
-       "importName": "CensusPEP",
-       "provenanceDomain": "census.gov"
+       "measurementMethod":  "CensusPEPSurvey",
+       "importName":  "CensusPEP",
+       "provenanceDomain":  "census.gov"
       },
       {
-       "val": {
-        "2011": 310346358,
-        "2012": 312855438,
-        "2013": 315219560,
-        "2014": 317746049,
-        "2015": 320098094,
-        "2016": 322087547,
-        "2017": 324473370,
-        "2018": 326289971
+       "val":  {
+        "2011":  310346358,
+        "2012":  312855438,
+        "2013":  315219560,
+        "2014":  317746049,
+        "2015":  320098094,
+        "2016":  322087547,
+        "2017":  324473370,
+        "2018":  326289971
        },
-       "measurementMethod": "CensusACS5yrSurvey",
-       "importName": "CensusACS5YearSurvey_AggCountry",
-       "provenanceDomain": "census.gov"
+       "measurementMethod":  "CensusACS5yrSurvey",
+       "importName":  "CensusACS5YearSurvey_AggCountry",
+       "provenanceDomain":  "census.gov"
+      },
+      {
+       "val":  {
```

Comment:
Git isn't rendering this well, but here the diffs are just adding CensusACS1yrSurvey